{
  "comments": [
    {
      "key": {
        "uuid": "3ac371cc_b9b5c3d1",
        "filename": "tools/releasetools/add_img_to_target_files.py",
        "patchSetId": 1
      },
      "lineNbr": 171,
      "author": {
        "id": 2936
      },
      "writtenOn": "2015-04-09T06:32:45Z",
      "side": 1,
      "message": "I thought this check would bypass actual creation.",
      "revId": "bd25b16df9c8844f36568be5cf75418b70bd74ec",
      "serverId": "1ec6b3db-b2c1-4fa4-84a2-4c7efe89ba71",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "3ac371cc_99f2876e",
        "filename": "tools/releasetools/add_img_to_target_files.py",
        "patchSetId": 1
      },
      "lineNbr": 171,
      "author": {
        "id": 10988
      },
      "writtenOn": "2015-04-09T07:52:38Z",
      "side": 1,
      "message": "It doesn\u0027t. \nIn my case, fs_type is \"\", which doesn\u0027t start with \"ext\", so it doesn\u0027t enter the if.",
      "parentUuid": "3ac371cc_b9b5c3d1",
      "revId": "bd25b16df9c8844f36568be5cf75418b70bd74ec",
      "serverId": "1ec6b3db-b2c1-4fa4-84a2-4c7efe89ba71",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "3ac371cc_397713d6",
        "filename": "tools/releasetools/add_img_to_target_files.py",
        "patchSetId": 1
      },
      "lineNbr": 171,
      "author": {
        "id": 2936
      },
      "writtenOn": "2015-04-09T08:21:40Z",
      "side": 1,
      "message": "Ohh, interesting. Your userdata is not EXT4?",
      "parentUuid": "3ac371cc_99f2876e",
      "revId": "bd25b16df9c8844f36568be5cf75418b70bd74ec",
      "serverId": "1ec6b3db-b2c1-4fa4-84a2-4c7efe89ba71",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "3ac371cc_4eb0448c",
        "filename": "tools/releasetools/add_img_to_target_files.py",
        "patchSetId": 1
      },
      "lineNbr": 171,
      "author": {
        "id": 10984
      },
      "writtenOn": "2015-04-09T23:51:27Z",
      "side": 1,
      "message": "It is not a question of what fs_type is being used, but the fact that info_dict doesn\u0027t have that info at all (no \"data_extra\" key).",
      "parentUuid": "3ac371cc_397713d6",
      "revId": "bd25b16df9c8844f36568be5cf75418b70bd74ec",
      "serverId": "1ec6b3db-b2c1-4fa4-84a2-4c7efe89ba71",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "3ac371cc_21acf9ec",
        "filename": "tools/releasetools/add_img_to_target_files.py",
        "patchSetId": 1
      },
      "lineNbr": 171,
      "author": {
        "id": 2936
      },
      "writtenOn": "2015-04-10T03:52:50Z",
      "side": 1,
      "message": "Can you paste me the build failure? I find it surprising tomato builds correctly with this, since it shouldn\u0027t have the data_extra key either.",
      "parentUuid": "3ac371cc_4eb0448c",
      "revId": "bd25b16df9c8844f36568be5cf75418b70bd74ec",
      "serverId": "1ec6b3db-b2c1-4fa4-84a2-4c7efe89ba71",
      "unresolved": false
    }
  ]
}